sources/documents/RTB/Verbali/Interni/VerbaleInterno_2024-04-25/capitolo-riunione.tex

```diff
@@ -0,0 +1,76 @@
+	\item Discussione delle attività svolte dai membri del gruppo durante la prima settimana di \glossario{sprint}.
+\end{itemize}
+
+\subsection{Attività svolte}
+\par All'inizio del meeting viene chiesto ai del gruppo di esporre le attività svolte durante la prima settimana di \glossario{sprint}.
+\subsubsection{\Programmatore}
+\par Vengono spiegate le tecnologie studiate e i test svolti su un \glossario{Dizionario Dati} di prova per comprendere il funzionamento dei modelli. Vengono inoltre esposti i dubbi e le difficoltà riscontrati, con un confronto tra gli altri membri per organizzare incontri e sistemare i problemi sorti. La seguente sezione è la più corposa poiché quella più discussa durante l'incontro.
+\begin{itemize}
+	\item Sono state riscontrate delle difficoltà iniziali nel lavorare ad un motore di database con esempi di database [SCLight], altrimenti non c'era modo di testare con [txtai].
+	\item L'idea iniziale è stata quella di prendere un database in formato \glossario{JSON} che potesse essere analizzato assieme alla \glossario{query} dell'utente e restituire il \glossario{prompt} con le parole prese dal \glossario{Dizionario Dati} e un punteggio in base all'accuratezza.
+	\item È stato tuttavia riscontrato un problema in [txtai] che coinvolgeva il formato \glossario{JSON}: infatti questo si presentava inadatto al modello che prediligeva quesiti posti in linguaggio naturale.
+	\item Così facendo [txtai] può trasformare la richiesta in un vettore che viene inserita in un indice di vettori.
+	\item Viene dato un suggerimento sulla distribuzione dei ruoli per il prossimo \glossario{sprint}, raccomandando più programmatori e mantenere fisso quello attuale, in modo da facilitare l'apprendimento delle tecnologie al membro nuovo nel ruolo e avere maggiore personale al lavoro sul risolvimento del problema.
+	\item Viene anche esposta la necessità di trovare un formato migliore per la \glossario{LLM} che verrà poi scelta, in modo che il linguaggio risulti meno tecnico per la \glossario{LLM} o trovare un modello che supporti la lettura del \glossario{Dizionario Dati} nel formato richiesto.
+	\item Infine viene teorizzato un sistema per sviluppare un \glossario{parser} che consenta di convertire il dizionario dati dal formato \glossario{JSON} ad un formato in linguaggio naturale. Questa viene inserita nelle attività da fare
+\end{itemize}
+
+\subsubsection{\Amministratore}
+\par Vengono spiegate le attività svolte come la stesura della struttura del \Glossario, [mancano parti che ha fatto Tommaso]. Viene inoltre puntualizzata la necessità di un incontro con il Professor. Tullio Vardanega per un chiarimento sullo stabilimento di un criterio col quale inserire i termini nel \Glossario. Del \Glossario si discute anche il tipo di uso (se esterno o interno) e il numero di occorrenze di termini che nei documenti vanno indicati come presenti nel documento. Questi dubbi verranno esposti al Diario di Bordo del 2024-04-26.
```

Comment:
```suggestion
\par Vengono spiegate le attività svolte come la stesura della struttura del \Glossario, [mancano parti che ha fatto Tommaso]. Viene inoltre puntualizzata la necessità di un incontro con il Professor. Tullio Vardanega per un chiarimento sullo stabilimento di un criterio col quale inserire i termini nel \Glossario. Relativamente al \Glossario\ si discute anche se il documento sia a uso interno o esterno, e il numero di occorrenze di termini che nei documenti vanno indicati come riferimento al \Glossario. Questi dubbi verranno esposti al Diario di Bordo del 26 aprile 2024.
```
da controllare la spaziatura dopo il comando `\Glossario` in nel caso di utilizzo all'interno della frase forse manca